temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
```

Comment:
Using `pdb` for debugging should be removed before committing the code. It is better to use logging for debugging purposes in production.
```
import logging

logging.basicConfig(level=logging.INFO)

logger = logging.getLogger(__name__)

logger.info('hello')
```